docs/eventing/brokers/broker-types/mt-channel-based-broker/README.md

```diff
@@ -0,0 +1,117 @@
+# Multi-tenant channel based Broker
+
+The Multi-tenant channel based Broker (MTChannelBasedBroker) uses [Channels](../../../channels) for event routing. It is shipped by default with Knative Eventing.
+Users should prefer native Broker implementations over the MTChannelBasedBroker and channel combo because it is usually more efficient. 
```

Comment:
Why should users prefer this? Please explain it if we're mentioning it here, i.e. how is it more efficient, and what is native referring to?
Please don't shorten words like "combo", write out "combination"

Reply:
Added a bit more context. This is mostly from the native Broker implementation pages.

Reply:
Is it clearer now?